turbinia/output_manager.py

```diff
@@ -172,12 +172,17 @@ class OutputWriter(object):
 
   NAME = 'base_output_writer'
 
-  def __init__(self, unique_dir=None, local_output_dir=None):
```

Comment:
I think it's worth adding a docstring here, to mention the auto-setting of some of these params

Reply:
Done.